### PR TITLE
add validation for webhook example

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
add configuration validation function to webhook example application.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: examples/customresourceinterpreter/webhook/app/options/validation_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+)
+
+func TestOptions_Validate(t *testing.T) {
+	rootPath := field.NewPath("Options")
+	tests := []struct {
+		name     string
+		opts     *Options
+		expected field.ErrorList
+	}{
+		{
```
Comment (XiShanYongYe-Chang) [inline examples/customresourceinterpreter/webhook/app/options/validation_test.go:17]: I think a normal case can be added.

Comment (gy95) [reply examples/customresourceinterpreter/webhook/app/options/validation_test.go:17]: fixed

**Comment by XiShanYongYe-Chang** [comment]
We maybe also need to enable examples test in `Makefile` like this:

https://github.com/karmada-io/karmada/blob/3a8c15ccfdc2d7be7e9aefd61157425e2f575659/Makefile#L104-L106

**Comment by gy95** [comment]
> We maybe also need to enable examples test in `Makefile` like this:
> 
> https://github.com/karmada-io/karmada/blob/3a8c15ccfdc2d7be7e9aefd61157425e2f575659/Makefile#L104-L106

done. 👍

**Pull request merged**